implementations/lips/head/Dockerfile

```diff
@@ -12,4 +12,5 @@ RUN apt-get update && apt-get -y --no-install-recommends install \
 COPY --from=build /usr/local/ /usr/local/
 COPY scheme-banner /usr/local/bin/
```

Comment:
`COPY failed: file not found in build context or excluded by .dockerignore: stat scheme-banner: file does not exist`